.github/workflows/ci.yml

```diff
@@ -70,12 +70,59 @@ jobs:
         with:
           fail_ci_if_error: false
 
-  molecule:
+  molecule-lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ['3.7']
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          path: ansible_collections/kubernetes/core
+
+      - name: Set up KinD cluster
+        uses: engineerd/setup-kind@v0.5.0
+
+      - name: Set up Python ${{ matrix.python_version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python_version }}
+
+      # The 3.3.0 release of molecule introduced a breaking change. See
```

Comment:
I think this has actually been fixed. We should confirm, but we can probably remove this restriction.